lib/handlers/injector-sor.ts

```diff
@@ -351,6 +369,8 @@ export abstract class InjectorSOR<Router, QueryParams> extends Injector<
                 ),
                 new NodeJSCache(new NodeCache({ stdTTL: 15, useClones: false }))
               ),
+              new NodeJSCache(new NodeCache({ stdTTL: 15, useClones: false }))
```

Comment:
little confused why do we need an extra cache here?

Reply:
There was a merge conflict with the hot fix stuff we just did https://github.com/Uniswap/routing-api/pull/363. We shouldn't need another in-memory cache. The only in-memory cache we added is for the TokenPropertiesProvider.

Reply:
makes sense!